<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CipherWindow</class>
 <widget class="QMainWindow" name="CipherWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>323</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CipherWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QPushButton" name="pbDecrypt">
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="sbMesh"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pbEncrypt">
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="sbPhaseShift"/>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="cbBaseSym"/>
    </item>
    <item row="1" column="0" rowspan="2" colspan="4">
     <widget class="QTextEdit" name="teMessage"/>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="sbShift"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Set Key:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="pbClear">
      <property name="text">
       <string>ClearAll</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
